src/core/Chip/Chip.md

```diff
@@ -0,0 +1,30 @@
+```js
+import { Chip } from 'suomifi-ui-components';
+
+const removeAction = () => {
+  alert('Selection removed');
+};
+
+<>
+  <div styles={{ display: 'inline' }}>
```

Comment:
This ends up in DOM as styles=[object, object]

Reply:
Ah, it doesn't seem to be necessary after all. Just the div is enough 👍

Reply:
Should now be fixed.